src/color-picker/color-picker.scss

```diff
@@ -6,10 +6,10 @@
 
 $iui-color-picker-content-width: ($iui-l * 9) + ($iui-s * 8); // (9 swatches per row) + (gaps)
 $iui-color-picker-max-width: ($iui-l * 10) + ($iui-s * 9); // (10 swatches per row) + (gaps)
-$iui-color-swatch-border-radius: 5px;
-$iui-inset-box-shadow: inset 0 0 0 1px rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
-$iui-active-box-shadow: 0 0 0 $iui-xxs rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-$iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+$iui-color-swatch-border-radius: $iui-border-radius * 2;
```

Comment:
That's a sneaky change. Do we really want it?

Reply:
It makes sense to me to have the color swatch border radius based off of `$iui-border-radius`.  It's a pretty small visual change.

Reply:
Feels too round to me personally. But if you really want to change it, I'd recommend doing it in a separate PR.

Reply:
Reverted back to 5px.